app/org/forms.py

```diff
@@ -0,0 +1,35 @@
+from flask_wtf import Form
+from wtforms import ValidationError
+from wtforms.ext.sqlalchemy.fields import QuerySelectMultipleField
+from wtforms.fields import (StringField, SubmitField, TextAreaField,
+                            IntegerField)
+from wtforms.fields.html5 import EmailField, URLField
+
+from .. import db
+from ..models import Organization, Tag
+
+
+class MultipleFileUploadField(StringField):
+    pass
+
+
+class OrganizationForm(Form):
+    name = StringField('Organization Name')
+    email = EmailField('Organization Contact Email')
+    phone = IntegerField('Phone number (eg 2108619271)')
+    address = StringField('Organization Address')
+    website_link = URLField('Organization website address \
+            (eg http://hack4impact.org)')
+    hours = StringField('Organization Hours of Operation')
+    description = TextAreaField('Description of your organization')
+    tags = QuerySelectMultipleField(
```

Comment:
this should be structured in a different way: Each Tag Type gets its own field, and the user can choose the tags within those types. So, one field each for ResourceType, Age Group, and Disability Services, and then the users can choose among the tags that are that tag type.

Reply:
ohhh i see. So like i should have one field  for each of those things, gotcha.  considering that adding tags + tag logic weren't part of this task, can we just merge this in as placeholder?

Reply:
yeah, just add in a TODO comment?